src/HttpRequestHandler.cpp

```diff
@@ -25,6 +28,33 @@ HttpResponse HttpRequestHandler::handleRequest(HttpRequest input)
 	return HttpResponse(ExceptionManager::getErrorStatus(InternalException("Something went wrong")), "");
 }
 
+bool HttpRequestHandler::findCgi(std::string uri)
+{
+	size_t found = uri.find("/cgi_bin");
+	if (found != std::string::npos)
+	{
+		bool ret = validateCgi(uri);
+		return ret;
+	}
+	return false;
+}
+
+bool HttpRequestHandler::validateCgi(std::string uri)
```

Comment:
Maybe the file validating would belong to the FileHandler rather than here? dunno